CMakeLists.txt

```diff
@@ -97,7 +97,7 @@ target_include_directories(${LIB_NAME}
 ###
 ### Tests
 ###
-
+enable_testing()
```

Comment:
Why is this change needed?  It seems problematic, since with this change, `enable_testing()` is not called before `lib/hpke/test/CMakeLists.txt` is processed (and the other library test files).  I would prefer to keep the `enable_testing()` directives in the test directories, where they are now.

Reply:
Maybe `enable_testing()` should be moved before first `add_diretcory()` call. My idea was to have a separate target that will invoke running unit-tests instead of moving to a separate forlder and running `ctest` there.